test_data/yast/xfs/xfs_partition.yaml

```diff
@@ -0,0 +1,37 @@
+---
+table_type: gpt
+partition_table_disk: vda
+disks:
+  - name: vda
+    allowed_unpartitioned: 0.00GB
+    partitions:
+      - size: 9250Mib
```

Comment:
From the code of `console/validate_fs_table` looks like we don't really validates size of the partitions. I guess you were confused with same parameters in `msdos` yaml. I guess we can skip it for now, as we should implement solution which ignores rounding errors due to block size, etc.

Reply:
Yes, please.